examples/vgg/caffee2npz_vgg_16.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+
+import chainer
+import chainer.links.caffe.caffe_function as caffe
+
+from chainercv.links import VGG16
+
+"""
+Please download a weight from here.
+http://www.robots.ox.ac.uk/%7Evgg/software/very_deep/
+caffe/VGG_ILSVRC_16_layers.caffemodel
+"""
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('caffemodel')
+    parser.add_argument('output')
+    args = parser.parse_args()
+
+    caffemodel = caffe.CaffeFunction(args.caffemodel)
+    model = VGG16(pretrained_model=None, n_class=1000)
+
+    model.conv1_1.conv.copyparams(caffemodel.conv1_1)
+    # The pretrained weights are trained to accept BGR images.
+    # Convert weights so that they accept RGB images.
+    model.conv1_1.conv.W.data[:] = model.conv1_1.conv.W.data[:, ::-1]
+    model.conv1_2.conv.copyparams(caffemodel.conv1_2)
```

Comment:
We can save `CaffeFunction` directly. Why do you copy the paramerters from `CaffeFunction` to `VGG16`?